util/random_test.go

```diff
@@ -8,11 +8,11 @@ import (
 func TestGetRandomTime(t *testing.T) {
 	t.Parallel()
 
-	testCases := []struct{
+	testCases := []struct {
 		lowerBound time.Duration
 		upperBound time.Duration
 	}{
-		{ 1 * time.Second, 10 * time.Second },
+		{1 * time.Second, 10 * time.Second},
```

Comment:
Another `gofmt` oddity: for a lot of arithmetic, it removes spaces (e.g. `max-min` in the file above), but here, it leaves them. I prefer it with spaces, but it's weird that it would be inconsistent.